index.html

```diff
@@ -6,60 +6,62 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
+    <script src="login.js"></script>
 </head>
 
 <body>
     <main>
         <div class="container">
             <div class="container__left">
-                <div class="info__area">
-                    <p class="info__area--h1">
+                <div class="info-area">
+                    <p class="info-area__h1">
```

Comment:
Optional: Fyi, we can also use variant classes here as `info-txt--h1`, `info-txt--h2`, `info-txt--text` etc.